docs/src/main/paradox/config-checker.md

```diff
@@ -0,0 +1,413 @@
+```
+Secure cookie is not a proper security solution. It is deprecated in Akka 2.4.x. Related config properties: [akka.remote.require-cookie = on, akka.remote.secure-cookie = abc]. Corresponding default values: [akka.remote.require-cookie = off, akka.remote.secure-cookie = ]. You may disable this check by adding [secure-cookie] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+### retry-gate-closed-for
+
+```
+Remote retry-gate-closed-for of [100 ms] is probably too short to be meaningful. This setting controls how much time should be elapsed before reattempting a new connection after a failed outbound connection. Setting it to a short interval may result in a storm of reconnect attempts.  Related config properties: [akka.remote.retry-gate-closed-for = 100ms]. Corresponding default values: [akka.remote.retry-gate-closed-for = 5 s]. You may disable this check by adding [retry-gate-closed-for] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+```
+Remote retry-gate-closed-for of [11000 ms] is probably too long. All messages sent to the gated address are dropped during the gating period. Related config properties: [akka.remote.retry-gate-closed-for = 11s]. Corresponding default values: [akka.remote.retry-gate-closed-for = 5 s]. You may disable this check by adding [retry-gate-closed-for] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+### prune-quarantine-marker-after
+
+```
+Don't change prune-quarantine-marker-after to a small value to re-enable communication with quarantined nodes. Such feature is not supported and any behavior between the affected systems after lifting the quarantine is undefined. Related config properties: [akka.remote.prune-quarantine-marker-after = 3600s]. Corresponding default values: [akka.remote.prune-quarantine-marker-after = 5 d]. You may disable this check by adding [prune-quarantine-marker-after] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+### enabled-transports
+
+```
+[akka.remote.netty.udp] is not a recommended transport for remote actor messages in production. Related config properties: [akka.remote.enabled-transports]. You may disable this check by adding [enabled-transports] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+### hostname
+
+*If using the classic remoting:*
+
+```
+hostname is not defined, which means that *InetAddress.getLocalHost.getHostAddress* will be used to resolve the hostname. That can result in wrong hostname in some environments, such as "127.0.1.1". Define the hostname explicitly instead. On this machine *InetAddress.getLocalHost.getHostAddress* is [192.168.2.189]. Related config properties: [akka.remote.netty.tcp.hostname = ]. Corresponding default values: [akka.remote.netty.tcp.hostname = ]. You may disable this check by adding [hostname] to configuration string list akka.diagnostics.checker.disabled-checks.
+```
+
+*If using Artery:*
```

Comment:
remove above related to classic remoting

Reply:
removed